### PR TITLE
CUDOS-889 Table of All proposals

### PR DESCRIPTION
**Pull request by stefanpetkov90**
- Created the layout for the Proposals page
- Created the table for the proposals, currently filled with dummy data

### CODE REVIEW

File: src/containers/Proposals/index.tsx [hunk 1]
```diff
@@ -1,10 +1,72 @@
...
-    </Box>
+    <>
+      <Box style={styles.stickyHeader}>
+        <Typography style={styles.headerStyle}>Proposals</Typography>
+        <Typography
+          style={styles.subheaderStyle}
+          variant="subtitle1"
+          color="text.secondary"
+        >
+          Here you can see the existing proposals’ statuses or create new one
```
Comment (SpaghettiOverload) [inline src/containers/Proposals/index.tsx:19]: I'd suggest changing it to: "Here you can create a new Proposal or check existing proposals status"

File: src/containers/Proposals/Proposal.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Box, Typography, Button, Avatar, Tooltip } from '@mui/material'
+import Card from 'components/Card/Card'
+import TestAvatar from '../../assets/vectors/test-avatar-sm.svg'
```
Comment (stanfilip0v) [inline src/containers/Proposals/Proposal.tsx:4]: We can now directly import from 'assets/vectors/...' no need to go back in the folders

File: src/containers/Proposals/Proposal.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
+import React from 'react'
```
Comment (stanfilip0v) [inline src/containers/Proposals/Proposal.tsx:1]: We can skip the import of React

File: src/containers/Proposals/index.tsx [hunk 4]
```diff
@@ -1,10 +1,72 @@
-import { Box, Typography } from '@mui/material'
+import React from 'react'
```
Comment (stanfilip0v) [inline src/containers/Proposals/index.tsx:1]: We can skip the import of React

File: src/containers/Proposals/index.tsx [hunk 5]
```diff
@@ -1,10 +1,72 @@
...
+import { Box, InputAdornment, Typography, Chip, Button } from '@mui/material'
+import Card from 'components/Card/Card'
+import SearchIcon from '../../assets/vectors/search-icon.svg'
```
Comment (stanfilip0v) [inline src/containers/Proposals/index.tsx:4]: We can now directly import from 'assets/vectors/...' no need to go back in the folders

File: src/containers/Proposals/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,124 @@
+import { styled, Input } from '@mui/material'
+import theme from '../../theme/index'
```
Comment (stanfilip0v) [inline src/containers/Proposals/styles.tsx:2]: We can now directly import from 'theme' no need to go back in the folders

**Review by SpaghettiOverload — APPROVED** [review]

File: src/containers/Proposals/index.tsx (hunk 1 above)
Comment (stefanpetkov90) [reply src/containers/Proposals/index.tsx:19]: I will discuss it with Eli and Petyo, because this is from the approved design.

**Review by stanfilip0v — APPROVED** [review]

**Pull request merged**